renovate.json

```
@@ -49,7 +49,7 @@
       },
       {
         "packageNames": ["styled-components"],
-        "allowedVersions": [">5.2.0"]
+        "allowedVersions": "!/5\\.2\\.0$/"
```

[nits] 万全を期すのであれば、キャレットがあったほうが良いかも？無くてもほぼ問題ないはずなので判断はお任せします。
```suggestion
        "allowedVersions": "!/^5\\.2\\.0$/"
```

---

すいませんこれGit上のtag名が `v5.2.0` なので、そっちを拾ってるかもしれないと思って省略してました！
`15.2.0` がきたら詰まりますが、その頃までには本指定まるまる外すつもりなので現状ママでもよいかと思っておりますがいかがでしょう？

---

例えv15がきても`15.2.1`からは入るのでこのままで問題ないかなと思います！